<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionWidgetVerticalSimpleLayout</class>
 <widget class="QWidget" name="AcquisitionWidgetVerticalSimpleLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>302</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AcquisitionWidgetVerticalSimple</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="acq_gbox">
     <property name="maximumSize">
      <size>
       <width>32767</width>
       <height>32767</height>
      </size>
     </property>
     <property name="title">
      <string>Acquisition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="num_images_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Number of wedges:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="num_images_cbox">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="angle_per_frame_label">
        <property name="text">
         <string>Range per frame (°):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="angle_per_frame_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="osc_start_cbox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Oscillation start (°):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="osc_start_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="osc_range_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Range per wedge (°):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="osc_range_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="exp_time_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Exposure time (s):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="exp_time_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="detector_roi_mode_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Detector mode:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="detector_roi_mode_combo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>9M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4M</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="kappa_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Kappa (°):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="kappa_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="kappa_phi_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Phi (°):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="kappa_phi_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="energy_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Energy (KeV):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="energy_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="dose_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Estimated dose (MGy):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="dose_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="resolution_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Resolution (Å):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="resolution_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="detector_distance_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Detector distance (mm):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="detector_distance_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="transmission_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Transmission (%):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="transmission_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="flux_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Flux (ph/s):</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="flux_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QCheckBox" name="check_alignment">
        <property name="text">
         <string>Check alignment</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="uncertainty_label">
        <property name="text">
         <string>Scan length (mm):</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLineEdit" name="uncertainty_ledit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>27</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>0.1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
